src/software/new_geom/segment.h

```diff
@@ -0,0 +1,88 @@
+#pragma once
+
+#include "software/new_geom/point.h"
+
+// This is a direct copy of geom/segment.h, this is needed to unblock work on the new_geom
+// shape hierarchy.
+// TODO: Refactor this to fit the new_geom hierarchy
```

Comment:
Is there a ticket for this? If not, let's file on, and either way, we should note the ticket here (ex. `TODO: Issue #1234`)

Reply:
Made issue #1098 and referenced it in the TODO, done in latest update